### PR TITLE
Added some notes on usage and Aldryn Boilerplates

### PR DESCRIPTION
**Pull request by evildmp**
* multiple Boilerplate support
* tidied some naming

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -26,9 +26,9 @@ Documentation
     :maxdepth: 2
 
     general/index
+    usage/index
```
Comment (FinalAngel) [inline docs/index.rst]: please add usage within index, would make more sense

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #############################
-Aldryn Boilerplate Bootstrap3
+Aldryn Boilerplate Bootstrap 3
 #############################
```
Comment (FinalAngel) [inline CHANGELOG.rst]: need to add additional # or?

File: docs/usage/basic.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+though, because it will be off-putting to people who don't want to have to use it. A reusable application should have
+requirements that are as generic as possible, not based on a particular frontend framework.
+
+So, although it means a little more work for you, you should **also** provide more generic frontend (templates, CSS
+etc) support for the application, and if you like, for other Boilerplates too.
+
+At the very least, the developers who use your application will find it easier to create templates and static file for
+it that support their own frontend conventions if they they can start with simple ones.
+
+Aldryn Boilerplates
```
Comment (FinalAngel) [inline docs/usage/basic.rst]: why not ====== on top of it

File: docs/usage/basic.rst [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+In your project
+===============
+
+See :doc:`/structure/templates` for guidelines on how to set up your project templates so that
+they take advantage of what it has to offer.
+
+Fundamentally, if your project's templates inherit from the `base.html
+<https://github.com/aldryn/aldryn-boilerplate-bootstrap3/blob/master/templates/base.html>`_ template, they'll be
+furnished with the classes, elements, hooks and other things they need.
+
```
Comment (FinalAngel) [inline docs/usage/basic.rst]: should have additional spacer, done like this verywhere

File: docs/index.rst [hunk 5]
```diff
@@ -26,9 +26,9 @@ Documentation
...
     guidelines/index
     structure/index
     testing/index
     codestyle/index
-    tips/index
```
Comment (FinalAngel) [inline docs/index.rst]: why did you remove tips?

File: docs/index.rst (hunk 1 above)
Comment (evildmp) [inline docs/index.rst]: Sorry, not sure what you mean here,

File: docs/usage/basic.rst (hunk 3 above)
Comment (evildmp) [inline docs/usage/basic.rst]: This section is a subsection of In your applications, so that's correct.

File: docs/index.rst (hunk 5 above)
Comment (evildmp) [inline docs/index.rst]: It's now inside usage - because they are tips on usage!

File: docs/index.rst (hunk 1 above)
Comment (FinalAngel) [inline docs/index.rst]: can you move the `usage` section to general? so we have "General > Usage" after installation or so

File: docs/index.rst (hunk 5 above)
Comment (FinalAngel) [inline docs/index.rst]: I'd still keep them separate as we will get general frontend tips inside there too

**Comment by FinalAngel** [comment]
thank you very much!
